pkg/controller/config/config.go

```diff
@@ -0,0 +1,236 @@
+/* Copyright © 2020 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+package config
+
+import (
+	"fmt"
+	"net"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/cluster-network-operator/pkg/render"
+	"gopkg.in/yaml.v2"
+
+	operatorv1 "github.com/ruicao93/antrea-operator/pkg/apis/operator/v1"
+	"github.com/ruicao93/antrea-operator/pkg/types"
+)
+
+func FillDefaults(clusterConfig *configv1.Network, operConfig *operatorv1.AntreaInstall) error {
```

Comment:
Looks like this function not only fill defaults, but also configs from network CRD. How about rename it as FillConfigs

Reply:
Done.